site/gdocs/pull-quote.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { OwidArticleBlock } from "./gdoc-types.js"
+
+export default function PullQuote({ d }: { d: OwidArticleBlock }) {
+    return (
+        <blockquote className={"pullQuote"}>
+            {d.value.map((d: any) => d.value).join("\n")}
```

Comment:
both d.value refer to two different things, that might be confusing. Could the inner `d` be typed?